report/src/sections/04-theory/01-abstract-domains.tex

```diff
@@ -15,7 +15,7 @@ \subsubsection{Abstract strings as regular expressions}\label{subsubsec:abstract
 \subsubsection{Abstract integers as union intervals}\label{subsubsec:abstract_domains_numbers} We have chosen a finite union and intersection of intervals to represent the abstract domain of the family of SQL number data types.
 We call them union of intervals as all such objects can be written as a union of intervals.
 We only consider integers, but we imagine that the domain can be easily extended to the reals.
-Let $\mathbf{INT}$ be the set of union intervals, that is inductively defined as:
+Let $\mathbf{INT}$ be the set of union intervals, and $\mathscr{I}$ be a set of intervals, that is inductively defined as:
```

Comment:
Skriv `Let $\mathbf{INT}$ be the set of union intervals, with members denoted $\mathscr{I}$, that is inductively defined as:`

Reply:
\mathscr{I} er ikke set af intervaller, det er et matematisk objekt.